pkg/libcose/Makefile.include

```diff
@@ -10,3 +10,7 @@ endif
 
 # Declare pseudomodules here to be selfcontained
 PSEUDOMODULES += libcose_crypt_%
+
+# There is a problem with the LLVM assembler and the assembly part of this
+# package
+TOOLCHAINS_BLACKLIST += llvm
```

Comment:
I do not get errors for `BOARD=iotlab-m3` or `BOARD=native` using `TOOLCHAIN=llvm` for `unittests`…

Reply:
Mhhh.... I do get it. Are you in the right branch ;-)?

Reply:
This PR branch yes.

Reply:
I of course get the warning for the BLACKLISTED toolchain.

Reply:
Ah! Mine are exactly as the [murdock ones](https://ci.riot-os.org/RIOT-OS/RIOT/9398/b30bc6725d6f9ff4307275a26e754dde5904c27d/output/compile/tests/unittests/samr21-xpro:llvm.txt)